### PR TITLE
[ALT] Move release goals content out of "Compatible Implementation" section

### PR DESCRIPTION
**Pull request by dblevins**
Alternative to #532 adds a header between the goals and the release artifacts.  For discussion by the Specification Committee

### CODE REVIEW

File: cdi/4.0/_index.md [hunk 1]
```diff
@@ -6,6 +6,22 @@ summary: "Release for Jakarta EE 10"
...
+* Java SE version will be aligned with the Core profile requirements regarding Java SE versions.
+* Consider addressing open issues:
+    * [Executable methods](https://github.com/eclipse-ee4j/cdi/issues/460)
+    * [Removal of deprecated bits](https://github.com/eclipse-ee4j/cdi/issues/472)
+    * [Improve constructor injection](https://github.com/eclipse-ee4j/cdi/issues/464)
+    * [AFTER_SUCCESS Observers and Events fired from within EntityListeners](https://github.com/eclipse-ee4j/cdi/issues/467)
+    * [Add methods to BeanConfigurator for applying decorators](https://github.com/eclipse-ee4j/cdi/issues/459)
+    * [All issues with a Lite label](https://github.com/eclipse-ee4j/cdi/issues?q=is%3Aissue+label%3ALite)
+
+# Downloads
```
Comment (ivargrimstad) [inline cdi/4.0/_index.md:23]: # Details

**Review by ivargrimstad — COMMENTED** [review]
+1

File: cdi/4.0/_index.md [hunk 2]
```diff
@@ -6,6 +6,22 @@ summary: "Release for Jakarta EE 10"
 
 Jakarta Contexts Dependency Injection specifies a means for obtaining objects in such a way as to maximize reusability, testability and maintainability compared to traditional approaches such as constructors, factories, and service locators (e.g., JNDI).
 
+The Jakarta Contexts and Dependency Injection 4.0 goals for this release are:
+
+* Split the existing 3.0 specification into pieces that separate out requirements based on the target runtime container support; Java SE, Jakarta Servlet, Jakarta Message Service, etc.
+* Breakup the TCK to separate out the concers along the lines of the target runtime requirements as outlined in 1.
+* Introduce a new CDI-lite specification that targets supporting build time compliation of applications.
```
Comment (Emily-Jiang) [inline cdi/4.0/_index.md:13]:
```suggestion
* Introduce a new CDI-lite section that targets supporting build time compliation of applications.
```

File: cdi/4.0/_index.md [hunk 3]
```diff
@@ -6,6 +6,22 @@ summary: "Release for Jakarta EE 10"
 
 Jakarta Contexts Dependency Injection specifies a means for obtaining objects in such a way as to maximize reusability, testability and maintainability compared to traditional approaches such as constructors, factories, and service locators (e.g., JNDI).
 
+The Jakarta Contexts and Dependency Injection 4.0 goals for this release are:
+
+* Split the existing 3.0 specification into pieces that separate out requirements based on the target runtime container support; Java SE, Jakarta Servlet, Jakarta Message Service, etc.
```
Comment (Emily-Jiang) [inline cdi/4.0/_index.md:11]: The main thing is to define CDI Lite portion from CDI Full, which enables build time compilation support.

**Comment by dblevins** [comment]
@Emily-Jiang Are you ok to do any changes to the content itself as a separate PR?  Perhaps working with the rest of the CDI project to come up with something everyone agrees on?

I'd like to keep my changes limited to structure/format and avoid any perception the Specification Committee is changing the goals/wording of their release.

**Review by ivargrimstad — APPROVED** [review]
Thanks David!

**Comment by ivargrimstad** [comment]
Replaced by https://github.com/jakartaee/specifications/pull/573

**Pull request closed without merging**